api/src/main/java/uk/gov/dhsc/htbhf/claimant/EmailMessageConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.dhsc.htbhf.claimant.message.payload.EmailType;
+
+import java.util.Map;
+
+@EnableConfigurationProperties
+@Configuration
+@ConfigurationProperties(prefix = "notify")
+@Getter
+@Setter
```

Comment:
In case you're wondering why this is here, the `Map` doesn't get populated without a getter and setter.

Reply:
Minor, but `@Getter` and `@Setter` could maybe go on the Map itself

Reply:
Ok, will do this on the next PR to save some build time.